### PR TITLE
OCPBUGS#10862: Update the cluster autoscaler information

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.10+

Issue:  https://issues.redhat.com/browse/OCPBUGS-10862

Link to docs preview: https://57995--docspreview.netlify.app/openshift-enterprise/latest/machine_management/applying-autoscaling.html#cluster-autoscaler-about_applying-autoscaling

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/cluster-autoscaler-about.adoc [hunk 1]
```diff
@@ -58,6 +58,13 @@ The cluster autoscaler takes pod priorities into account. The Pod Priority and P
 
 Pods with priority lower than the cutoff value do not cause the cluster to scale up or prevent the cluster from scaling down. No new nodes are added to run the pods, and nodes running these pods might be deleted to free resources.
 
+The cluster autoscaling is supported on the following cloud providers:
```
Comment (sunzhaohua2) [inline modules/cluster-autoscaler-about.adoc:61]: I am not sure if we really need this list,  as long as support machine-api, it also supports autoscaler, if we must list this, gcp, vsphere, ibmcloud, nutanix all these platforms support cluster autoscaler.

File: modules/cluster-autoscaler-about.adoc [hunk 2]
```diff
@@ -29,7 +29,7 @@ Ensure that the `maxNodesTotal` value in the `ClusterAutoscaler` resource defini
 Every 10 seconds, the cluster autoscaler checks which nodes are unnecessary in the cluster and removes them. The cluster autoscaler considers a node for removal if the following conditions apply:
 
 * The node utilization is less than the _node utilization level_ threshold for the cluster. The node utilization level is the sum of the requested resources divided by the allocated resources for the node. If you do not specify a value in the `ClusterAutoscaler` custom resource, the cluster autoscaler uses a default value of `0.5`, which corresponds to 50% utilization.
-* The cluster autoscaler can move all pods running on the node to the other nodes.
+* The cluster autoscaler can move all pods running on the node to the other nodes. The scheduler is responsible to schedule pods on the nodes.
```
Comment (elmiko) [inline modules/cluster-autoscaler-about.adoc:32]: would it help to refine this even more?

```suggestion
* The cluster autoscaler can move all pods running on the node to the other nodes. The Kubernetes scheduler is responsible to schedule pods on the nodes.
```

File: modules/cluster-autoscaler-about.adoc (hunk 1 above)
Comment (elmiko) [reply modules/cluster-autoscaler-about.adoc:61]: i agree, we should specify that autoscaling works where the machine-api is present. this covers all our existing platforms and allows us to talk about future platforms that will be added to the machine-api without the need to keep a specific list.

**Review by elmiko — COMMENTED** [review]
just a couple thoughts

File: modules/cluster-autoscaler-about.adoc (hunk 2 above)
Comment (xenolinux) [reply modules/cluster-autoscaler-about.adoc]: Addressed

File: modules/cluster-autoscaler-about.adoc [hunk 3]
```diff
@@ -58,6 +58,8 @@ The cluster autoscaler takes pod priorities into account. The Pod Priority and P
 
 Pods with priority lower than the cutoff value do not cause the cluster to scale up or prevent the cluster from scaling down. No new nodes are added to run the pods, and nodes running these pods might be deleted to free resources.
 
+The cluster autoscaling is supported for the platforms that have machine API available on it.
```
Comment (xenolinux) [inline modules/cluster-autoscaler-about.adoc:61]: @elmiko Does this sentence look good? can we say "supported"?

Comment (elmiko) [reply modules/cluster-autoscaler-about.adoc:61]: i think that's accurate to me

Comment (xenolinux) [reply modules/cluster-autoscaler-about.adoc:61]: Cool. Thanks

**Comment by sunzhaohua2** [comment]
LGTM

File: modules/cluster-autoscaler-about.adoc (hunk 3 above)
Comment (opayne1) [inline modules/cluster-autoscaler-about.adoc:61]:
```suggestion
Cluster autoscaling is supported for the platforms that have machine API available on it.
```

**Review by opayne1 — COMMENTED** [review]
Just a few small comments from me

**Review by opayne1 — COMMENTED** [review]
Just on comment before I can merge. Sorry about that!

**Pull request merged**